packages/@aws-cdk/aws-lambda-python/README.md

```diff
@@ -145,6 +145,18 @@ new python.PythonFunction(this, 'function', {
 });
 ```
 
+In situations where files are needed for bundling that are from a different container, you can specify the container ID from which additional volumes should be mounted.
+
+ ```ts
+const entry = '/path/to/function';
+
+new python.PythonFunction(this, 'function', {
+  entry,
+  runtime: Runtime.PYTHON_3_8,
+  bundling: { volumesFrom: ['777f7dc92da7'] },
+});
+```
+
```

Comment:
Only realized now that this is in the python readme. Not sure that makes a whole lot of sense right now. I can exempt the PR for readme change as well.

Reply:
so i guess i also remove this now?